src/Service/CognitoIdentityProvider/composer.json

```
@@ -0,0 +1,30 @@
+{
+    "name": "async-aws/cognito-identity-provider",
+    "type": "library",
+    "description": "AsyncAws CognitoIdentityProvider client.",
+    "keywords": [
+        "aws",
+        "cognito-identity-provider"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.2.5",
+        "ext-json": "*",
+        "async-aws/core": "^0.5"
```

```suggestion
        "async-aws/core": "^1.0"
```

It will fixes the prefer-lowest test